src/rgw/rgw_rest_sts.cc

```diff
@@ -188,20 +189,20 @@ WebTokenEngine::get_from_jwt(const DoutPrefixProvider* dpp, const std::string& t
     if (error == -EACCES) {
       throw -EACCES;
     }
-    ldpp_dout(dpp, 5) << "Invalid JWT token" << dendl;
+    ldpp_dout(dpp, 5) << "Invalid JWT" << dendl;
```

Comment:
This debug statement looks exactly like the one on line 196.  Is the error code logged elsewhere?  If not, might be helpful to write it out here.